### PR TITLE
fix: more friendly error output when running rootless mode 

### PR DESCRIPTION
**Pull request by fengxsong**
1. more friendly error output when running rootless mode without required packages like `uidmap`/`fuse-overlayfs`
2. use builtin `cobra.Command.CommandPath()` to replace `getFullCmdName`

Signed-off-by: fengxsong <fengxsong@outlook.com>

### CODE REVIEW

File: cmd/sealos/cmd/root.go [hunk 1]
```diff
@@ -57,6 +58,8 @@ func init() {
 	_ = fs.MarkHidden("cluster-root")
 	fs.StringVar(&runtimeRootDir, "sealos-root", constants.DefaultRuntimeRootDir, "root directory for sealos actions")
 	_ = fs.MarkHidden("sealos-root")
+	// add unrelated command names that don't required buildah sdk.
+	buildah.AddUnrelatedCommandNames("docs", "exec", "gen", "scp", "version")
```
Comment (cuisongliu) [inline cmd/sealos/cmd/root.go:62]: add cert

File: cmd/sealos/cmd/root.go [hunk 2]
```diff
@@ -57,6 +58,8 @@ func init() {
 	_ = fs.MarkHidden("cluster-root")
 	fs.StringVar(&runtimeRootDir, "sealos-root", constants.DefaultRuntimeRootDir, "root directory for sealos actions")
 	_ = fs.MarkHidden("sealos-root")
+	// add unrelated command names that don't required buildah sdk.
+	buildah.AddUnrelatedCommandNames("cert", "docs", "exec", "gen", "scp", "version")
```
Comment (zzjin) [inline cmd/sealos/cmd/root.go:62]: Move this params to const ahead of this file?

File: pkg/buildah/buildah.go [hunk 3]
```diff
@@ -107,7 +108,7 @@ func RegisterGlobalFlags(fs *pflag.FlagSet) error {
 var (
 	globalFlagResults globalFlags
 	rootCmd           *cobra.Command
-	rootCmdName       string
+	unrelatedCommands = sets.NewString("version")
```
Comment (zzjin) [inline pkg/buildah/buildah.go:111]: `k8s.io/apimachinery/pkg/util/sets`'s `String` is deprecated. Use their new: `https://github.com/kubernetes/apimachinery/blob/master/pkg/util/sets/set.go`

Comment (zzjin) [reply pkg/buildah/buildah.go:111]: Maybe just new empty string is enough.(Suppress const vars.)

**Review by fanux — APPROVED** [review]

**Review by zzjin — CHANGES_REQUESTED** [review]
Need some update.

File: pkg/buildah/buildah.go (hunk 3 above)
Comment (fengxsong) [reply pkg/buildah/buildah.go:111]: OK

File: cmd/sealos/cmd/root.go (hunk 2 above)
Comment (fengxsong) [reply cmd/sealos/cmd/root.go:62]: Each subcommands can call function `buildah.AddUnrelatedCommandNames` separately, so I don't see the need to make it as constant. @cuisongliu what do you suggest?

**Review by zzjin — APPROVED** [review]

**Pull request merged**